test/test262.sh

```diff
@@ -20,10 +20,14 @@ TIMEOUT=${TIMEOUT:-10000}
 
 if [ "$(uname)" = 'Darwin' ]; then
   threads=$(sysctl -n hw.logicalcpu)
+  ((threads = threads * 2))
```

Comment:
It's interesting to me that this seems to speed things up as apparently the M1 chips don't have hyperthreading? https://apple.stackexchange.com/questions/425668/does-the-m1-chip-apple-silicon-use-hyper-threading-a-k-a-simultaneous-multi

Reply:
Ah good point. I'll experiment with unconditionally doubling this number and send a PR if it makes a big difference.

Reply:
I tried unconditionally doubling the threads on my workstation (from 11 to 22) and it didn't make much of a difference: 3m36s to 3m34s.